python-lib/dku_filtering/filtering.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+from functools import reduce
+
+
+def numerical_filter(df, filter):
```

Comment:
name.
```suggestion
def numerical_filter(df, filter_):
```
If possible, it is better not to use builtin names in functions. The general workaround is to use var name `filter_`, that is the main role of placing an underscore at the end of a var

Reply:
Agreed